codebuild/ancient-linux-x64.yml

```diff
@@ -0,0 +1,24 @@
+version: 0.2
+#this build spec assumes centos5 for building manylinux1 pypi packages
+# relevant packages on image:
+#     gcc 4.9
+#     cmake 3.5
+phases:
+  install:
+    commands:
+
+  pre_build:
+    commands:
+      - export CC=gcc
+  build:
+    commands:
+      - echo Build started on `date`
+      - mkdir build
+      - cd build
+      - cmake ../
```

Comment:
No generator? Does that work?

Reply:
yeah, make is the default.

It works everywhere except for windows.